src/main/java/com/linecorp/armeria/server/docs/EnumInfo.java

```diff
@@ -22,14 +22,15 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 
 import org.apache.thrift.meta_data.EnumMetaData;
 import org.apache.thrift.protocol.TType;
 
 class EnumInfo extends TypeInfo implements ClassInfo {
 
-    static EnumInfo of(EnumMetaData enumMetaData) {
+    static EnumInfo of(EnumMetaData enumMetaData, Map<String, String> docStrings) {
```

Comment:
It seems like `namespace` is used only when `docStrings` is used, so yes. To avoid code duplication, you might want to make `namespace` `@Nullable` so that the new factory method handles the legacy factory method as well.